auto-generated/api-testing/vfwcvt.c

```diff
@@ -460,3 +460,2283 @@ vfloat64m8_t test_vfwcvt_f_f_v_f64m8_m(vbool8_t mask, vfloat32m4_t src, size_t v
   return __riscv_vfwcvt_f_f_v_f64m8_m(mask, src, vl);
 }
 
+vfloat16mf4_t test_vfwcvt_f_x_v_f16mf4(vint8mf8_t src, size_t vl) {
+  return __riscv_vfwcvt_f_x_v_f16mf4(src, vl);
+}
+
+vfloat16mf2_t test_vfwcvt_f_x_v_f16mf2(vint8mf4_t src, size_t vl) {
+  return __riscv_vfwcvt_f_x_v_f16mf2(src, vl);
+}
+
+vfloat16m1_t test_vfwcvt_f_x_v_f16m1(vint8mf2_t src, size_t vl) {
+  return __riscv_vfwcvt_f_x_v_f16m1(src, vl);
+}
+
+vfloat16m2_t test_vfwcvt_f_x_v_f16m2(vint8m1_t src, size_t vl) {
+  return __riscv_vfwcvt_f_x_v_f16m2(src, vl);
+}
+
+vfloat16m4_t test_vfwcvt_f_x_v_f16m4(vint8m2_t src, size_t vl) {
+  return __riscv_vfwcvt_f_x_v_f16m4(src, vl);
+}
+
+vfloat16m8_t test_vfwcvt_f_x_v_f16m8(vint8m4_t src, size_t vl) {
+  return __riscv_vfwcvt_f_x_v_f16m8(src, vl);
+}
+
+vfloat16mf4_t test_vfwcvt_f_xu_v_f16mf4(vuint8mf8_t src, size_t vl) {
+  return __riscv_vfwcvt_f_xu_v_f16mf4(src, vl);
+}
+
+vfloat16mf2_t test_vfwcvt_f_xu_v_f16mf2(vuint8mf4_t src, size_t vl) {
+  return __riscv_vfwcvt_f_xu_v_f16mf2(src, vl);
+}
+
+vfloat16m1_t test_vfwcvt_f_xu_v_f16m1(vuint8mf2_t src, size_t vl) {
+  return __riscv_vfwcvt_f_xu_v_f16m1(src, vl);
+}
+
+vfloat16m2_t test_vfwcvt_f_xu_v_f16m2(vuint8m1_t src, size_t vl) {
+  return __riscv_vfwcvt_f_xu_v_f16m2(src, vl);
+}
+
+vfloat16m4_t test_vfwcvt_f_xu_v_f16m4(vuint8m2_t src, size_t vl) {
+  return __riscv_vfwcvt_f_xu_v_f16m4(src, vl);
+}
+
+vfloat16m8_t test_vfwcvt_f_xu_v_f16m8(vuint8m4_t src, size_t vl) {
+  return __riscv_vfwcvt_f_xu_v_f16m8(src, vl);
+}
+
+vint32mf2_t test_vfwcvt_x_f_v_i32mf2(vfloat16mf4_t src, size_t vl) {
```

Comment:
Should this have a rounding mode? Float to int loses decimal places which need to be rounded.

Reply:
Yes I think so. Thank you for catching the miss.